src/SGB_Command_System.md

```diff
@@ -24,6 +24,11 @@ after having displayed the picture). The Cancel Mask function may be
 also invoked (optionally) by completion of PAL_SET and ATTR_SET
 commands.
 
+:::tip
+
+SGB freezes/unfreezes the Game Boy screen automatically whenever the LCD is
+turned off/on.
```

Comment:
I think we're missing the closing of the box here. See https://github.com/gbdev/pandocs/blob/master/CONTRIBUTING.md#custom-containers

Reply:
Of course. I'm learning the syntax as I go.